environment/config.go

```diff
@@ -160,13 +160,13 @@ func (c Configuration) Configure(cfgParams ConfigurationParameters) (ConfigureRe
 
 		nsInitStatic := namespace.NewStaticInitializer(nsList)
 
-		shardSet, hostShardSets, err := newStaticShardSet(c.Static.TopologyConfig.Shards, c.Static.ListenAddress, cfgParams.HostID)
+		shardSet, hostShardSets, err := newStaticShardSet(c.Static.TopologyConfig.Shards, c.Static.TopologyConfig.Hosts)
 		if err != nil {
 			err = fmt.Errorf("unable to create shard set for static config: %v", err)
 			return emptyConfig, err
 		}
 		staticOptions := topology.NewStaticOptions().
-			SetReplicas(1).
+			SetReplicas(len(c.Static.TopologyConfig.Hosts)).
```

Comment:
This means each host is a replica of each other.  This isn't a good idea, ha.  If this is truly what we want though we should think about expressing it better in the config as this is not clear what's being done.

Reply:
So should we let users pass in `replicas` in the config and then if they don't specify, we just default to number hosts?